commit 43aba5ffffc6d35ed97db035c5818c76652b06de
Author: Matthias Reichl <hias@horus.com>
Date:   Mon Sep 28 23:55:13 2020 +0200

    move /etc/systemd/system to /storage/.config/system.d

diff --git a/meson.build b/meson.build
index dbbddb68e2..4592cd1094 100644
--- a/meson.build
+++ b/meson.build
@@ -255,7 +255,7 @@
 conf.set_quoted('SYSTEMD_TTY_ASK_PASSWORD_AGENT_BINARY_PATH', join_paths(rootbindir, 'systemd-tty-ask-password-agent'))
 conf.set_quoted('SYSTEMD_USERWORK_PATH',                      join_paths(rootlibexecdir, 'systemd-userwork'))
 conf.set_quoted('SYSTEMD_VERITYSETUP_PATH',                   join_paths(rootlibexecdir, 'systemd-veritysetup'))
-conf.set_quoted('SYSTEM_CONFIG_UNIT_DIR',                     join_paths(pkgsysconfdir, 'system'))
+conf.set_quoted('SYSTEM_CONFIG_UNIT_DIR',                     '/storage/.config/system.d')
 conf.set_quoted('SYSTEM_DATA_UNIT_DIR',                       systemunitdir)
 conf.set_quoted('SYSTEM_ENV_GENERATOR_DIR',                   systemenvgeneratordir)
 conf.set_quoted('SYSTEM_GENERATOR_DIR',                       systemgeneratordir)
